CHANGELOG.md

```diff
@@ -1,3 +1,68 @@
+GOBBLIN 0.9.0
+-------------
+
+### Created Date: 12/13/2016
+
+## Highlights
```

Comment:
I think the main change is for other projects that have a dependency on gobblin artifacts. So might be worth highlighting the change.